ocp_resources/virtual_machine_instance.py

```diff
@@ -204,11 +205,48 @@ def get_xml(self):
         Returns:
             xml_output(string): VMI XML in the multi-line string
         """
+        cmd = f"virsh {self.virt_launcher_pod_hypervisor_connection_uri} dumpxml {self.namespace}_{self.name}"
         return self.virt_launcher_pod.execute(
-            command=["virsh", "dumpxml", f"{self.namespace}_{self.name}"],
+            command=shlex.split(cmd),
             container="compute",
         )
 
+    @property
+    def virt_launcher_pod_user(self):
+        """
+        Get Virt Launcher Pod User value
+
+        Returns:
+            Int: Virt Launcher Pod runAsUser value
```

Comment:
UID?

Reply:
Was using the value as per spec. Updated, so that it's  more clear.